### PR TITLE
ROC-8030 RPA-claimant-response

### PR DESCRIPTION
**Pull request by shashisk97**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-8030

### Change description ###

Allow RPA robots to read the defendant's response when a claim has been issued against them

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

File: rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/ClaimantResponseJsonMapper.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        this.address = address;
+    }
+
+    public JsonObject map(Claim claim) {
+        ClaimantResponse response = claim.getClaimantResponse()
+            .orElseThrow(() -> new IllegalArgumentException("Missing response"));
+        String claimantEmail = claim.getSubmitterEmail();
+        LocalDateTime claimantRespondedAt = claim.getClaimantRespondedAt().orElse(null);
+        return new NullAwareJsonObjectBuilder()
+            .add("caseNumber", claim.getReferenceNumber())
```
Comment (AlokDatta) [inline rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/ClaimantResponseJsonMapper.java:30]: Can we move this keys to a constant please?

File: src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/ClaimantResponseNotificationService.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        requireNonNull(event);
+        EmailData emailData = prepareEmailData(event.getClaim());
+        emailService.sendEmail(emailProperties.getSender(), emailData);
+    }
+
+    private EmailData prepareEmailData(Claim claim) {
+        requireNonNull(claim);
+        EmailAttachment responsePDFAttachment = createClaimantResponseAttachment(claim);
+        return new EmailData(emailProperties.getClaimantResponseRecipient(),
+            "J claimant response " + claim.getReferenceNumber(),
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/ClaimantResponseNotificationService.java:48]: Move this String to a constant please

Comment (shashisk97) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/ClaimantResponseNotificationService.java]: made changes

File: rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/ClaimantResponseJsonMapper.java (hunk 1 above)
Comment (shashisk97) [reply rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/ClaimantResponseJsonMapper.java:30]: i have managed same standard as of existing mappers.

**Pull request closed without merging**